src/app/public/modules/radio/radio.component.ts

```diff
@@ -91,15 +101,42 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     return this._disabled;
   }
 
+/**
+ * Specifies an ID for the radio button.
+ * @default a unique, auto-incrementing integer. For example: `sky-radio-1`
+ */
   @Input()
   public id = `sky-radio-${++nextUniqueId}`;
 
+/**
+ * Defines an ARIA label for the radio button to support
+ * [accessibility](https://developer.blackbaud.com/skyux-forms/docs/radio#accessibility) when the
+ * radio button does not include a visible label. This property must be set if you are using an
+ * icon radio button. If the radio button includes a visible label on the screen, use the
+ * `labelledBy` property instead.
+ */
   @Input()
   public label: string;
 
+/**
+ * Sets the radio button's `aria-labelledby` attribute to support
+ * [accessibility](https://developer.blackbaud.com/skyux-forms/docs/radio#accessibility). The value
+ * should be the HTML element ID (without the leading #) of the element that labels the radio
+ * button. If the radio button does not include a visible label on the screen, use the `label` property instead.
+ */
   @Input()
   public labelledBy: string;
 
+// tslint:disable: max-line-length
+/**
+ * This property is deprecated in favor of
+ * [the `name` property on the `sky-radio-group element`](https://developer.blackbaud.com/skyux-forms/docs/radio#skyradiogroupcomponent).
+ * We recommend using the `sky-radio-group` element with all radio buttons, but if you opt not to,
+ * then this property specifies a name for a group of radio buttons.
+ * @deprecated
+ * @required
```

Comment:
This one is a little weird because of its history. Seems like just @deprecated would suffice here. Most cases will fall into the "radio-group" scenario, so it won't be required.

Reply:
✔️